CHANGELOG.md

```diff
@@ -1,3 +1,46 @@
+# 3.0.0 (2022-06-30)
+
+## :tada: Enhancements
+
+* Migrate to [AWS SDK for Go v2](https://github.com/aws/aws-sdk-go-v2).
+
+## :boom: Breaking changes
+
+* Bump minimum Go version from `1.14` to `1.15` as required by SDK V2
+* Driver constructor is changed to takes new type of `qldbSession` client. Application code need to be modified for [qldbSession client](https://github.com/aws/aws-sdk-go-v2/tree/main/service/qldbsession) construction.
```

Comment:
takes->take, need->needs

Reply:
better to change the doc link to https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/qldbsession